gui/ThemeEval.cpp

```diff
@@ -169,6 +169,14 @@ ThemeEval &ThemeEval::addSpace(int size) {
 	return *this;
 }
 
+#define SCALEVALUE(val) (val > 0 ? val * _scaleFactor : val)
```

Comment:
Most likely due to the fact that I'm not that much into the scummvm ui code - but why only for positive values?

Reply:
-1 is a special value, meaning "compute it"